src/Payload.php

```diff
@@ -411,7 +411,10 @@ public function jsonSerialize()
         if ($this->alert instanceof Alert) {
             $payload[self::PAYLOAD_ROOT_KEY]->{self::PAYLOAD_ALERT_KEY} = $this->alert;
         } elseif(is_string($this->alert)){
-            $payload[self::PAYLOAD_ROOT_KEY]->{self::PAYLOAD_ALERT_KEY} = json_decode($this->alert, true);
+            $json = json_decode($this->alert, true);
+            if($json){
+                $payload[self::PAYLOAD_ROOT_KEY]->{self::PAYLOAD_ALERT_KEY} = $json;
+            }
```

Comment:
We still miss the case when `$this->alert` is not a json, but just a string. In this case it will not be added to payload, but I think it should be added

Reply:
You want to add exception?

Reply:
> We still miss the case when `$this->alert` is not a json, but just a string. In this case it will not be added to payload, but I think it should be added

I understand that you want ^)
fixed